synapse/cores/common.py

```diff
@@ -2597,13 +2597,8 @@ def _tufosByIn(self, prop, valus, limit=None):
         return ret
 
     def _tufosByInetCidr(self, prop, valu, limit=None):
-
-        ipv4str, cidr = valu.split('/', 1)
-        ipv4addr, _ = s_datamodel.getTypeParse('inet:ipv4', ipv4str)
-        mask = (2 ** (32 - int(cidr)))
-        ipv4addr &= ~mask
-
-        return self.getTufosBy('range', prop, (ipv4addr, ipv4addr + mask), limit=limit)
+        lowerbound, upperbound = self.getTypeCast('inet:ipv4:cidr', valu)
```

Comment:
total nit, but the tuple return syntax used here and in the range call below causes the creation of an extra tufo object even though we're not using the separated values.  Ie
```
rang = self.getTypeCast(...
return self.getTufosBy('range', prop, rang, limit=limit)
```